bundles/core/org.eclipse.smarthome.core.thing.xml/src/main/java/org/eclipse/smarthome/core/thing/xml/internal/ThingTypeXmlProvider.java

```diff
@@ -43,9 +45,10 @@
  * used to merge with the {@link ThingTypeXmlResult} objects to create valid {@link ThingType} objects. After the merge
  * process has been finished, the cache is cleared again. The merge process is started when {@link #addingFinished()} is
  * invoked from the according {@link XmlDocumentBundleTracker}.
- *
+ * 
```

Comment:
You added a trailing whitespace here and I think @maggu2810 does not like this ;). Could you please apply save actions, to remove trailing white spaces? I think it also affects a lot other files.